run_all_tests.bat

```diff
@@ -50,8 +50,13 @@ echo;
 echo ---------------------------------------
 echo TESTING INSTRON Rec Sim
 REM call %PYTHON% "%EPICS_KIT_ROOT%\support\IocTestFramework\master\run_tests.py" -r -pf %MYPVPREFIX%  -d instron_stress_rig -p %EPICS_KIT_ROOT%\ioc\master\INSTRON\iocBoot\iocINSTRON-IOC-01
+echo ---------------------------------------
+echo;
 
 echo ---------------------------------------
+echo TESTING XYBEAMSTOP Dev Sim
+call %PYTHON% "%EPICS_KIT_ROOT%\support\IocTestFramework\master\run_tests.py" -pf %MYPVPREFIX%  -d xybeamstop -p %EPICS_KIT_ROOT%\ioc\master\GALIL\iocBoot\iocGALIL-IOC-01
```

Comment:
This command doesn't actually result in the tests running. I suspect because it doesn't launch a GALIL IOC so the test has nothing to act on.